apps/src/StudioApp.js

```diff
@@ -1631,6 +1654,39 @@ StudioApp.prototype.getUnfilledFunctionalBlock = function () {
 };
 
 /**
+ * @returns {string} The name of a function that doesn't have any examples, or
+ *   undefined if all have at least one.
+ */
+StudioApp.prototype.getFunctionWithoutExample = function () {
+  var definitionNames = Blockly.mainBlockSpace.getTopBlocks().filter(function (block) {
+    return block.type === 'functional_definition' && !block.isVariable();
+  }).map(function (definitionBlock) {
+    return definitionBlock.getProcedureInfo().name;
+  });
+
+  var exampleNames = Blockly.mainBlockSpace.getTopBlocks().filter(function (block) {
+    if (block.type !== 'functional_example') {
+      return false;
+    }
+
+    // Only care about functional_examples that have an ACTUAL input (i.e. it's
+    // clear which function they're for
+    var actual = block.getInputTargetBlock('ACTUAL');
+    return actual && actual.getTitleValue('NAME');
+  }).map(function (exampleBlock) {
+    return exampleBlock.getInputTargetBlock('ACTUAL').getTitleValue('NAME');
+  });
+
+  var exampleless;
+  definitionNames.forEach(function (def) {
```

Comment:
Oops, nevermind! No closure tools in this context, I've been in closure tools land for too long :). lodash versions of those could work, but not that important